docs/logging/harvester-logging.md

```diff
@@ -77,6 +77,16 @@ If desired, you can modify how many resources are dedicated to those components.
 
 ![](/img/v1.2/logging/modify-logging-fluent-resources.png)
 
+_Available as of v1.2.0_
+
+ The navigation path is changed to `Harvester > Advanced > Addons > rancher-logging`, other steps are same as above.
+
+![](/img/v1.2/logging/modify-logging-resources-from-addon.png)
```

Comment:
I think we could just replace the above steps with v1.2.0 designs since users can check the v1.1 doc if needed. WDUT @LucasSaintarbor